library/catalog/templates/base.html

```diff
@@ -77,6 +126,39 @@ <h5>Our social media</h5>
 
 
 <!-- Custom scripts -->
+<script type="text/javascript">
+    function login_user(){
+        email = document.getElementById("exampleInputUsername").value;
+        password = document.getElementById("exampleInputPassword").value;
+        csrfmiddlewaretoken = document.getElementsByName("csrfmiddlewaretoken")[0].value;
+        $("#errorlogin").html("");
+        $.ajax({
+            type:"POST",
+            url:'{% url 'func_login' %}',
+            data:{
+                'csrfmiddlewaretoken': csrfmiddlewaretoken,
+                'username':email,
+                'password':password,
+            },
+            success : function(data){
+                console.log(data);
```

Comment:
Do we really need to show reply in the console ?

Reply:
Okay, I will delete this